exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -0,0 +1,134 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.cloud.trace.v2.TraceServiceClient;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TruncatableString;
+import com.google.protobuf.BoolValue;
+import com.google.rpc.Status;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.trace.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.*;
+
+@RunWith(JUnit4.class)
+public class EndToEndTest {
```

Comment:
nit: I would recommend a ordering like this to improve readability:

final static variable
instance fields
 instance methods

Reply:
done; looks cleaner now for sure